stdlib/public/SwiftShims/RuntimeShims.h

```diff
@@ -84,6 +86,24 @@ __swift_size_t _swift_stdlib_getHardwareConcurrency(void);
 /// the Swift runtime and user binaries need to agree on this value.
 #define _swift_MinAllocationAlignment (__swift_size_t) 16
 
+struct ElfSectionInfo {
```

Comment:
This doesn't look ELF-specific, let's call this something more generic, maybe 'SectionDescriptor'?